NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
@johnament  I was told this is better to be kept as is. No real requirement to do it.. better to keep as is